src/JitsiApp.ts

```diff
@@ -40,10 +41,22 @@ export class JitsiApp extends App {
 		provider.limitTokenToRoom = await settings.getValueById(AppSetting.JitsiLimitTokenToRoom);
 		provider.tokenAuditor = await settings.getValueById(AppSetting.JitsiTokenAuditor);
 		provider.tokenExpiration = await settings.getValueById(AppSetting.JitsiTokenExpiration);
+		provider.useJaaS = await settings.getValueById(AppSetting.UseJaaS);
+		provider.jaasPrivateKey = await settings.getValueById(AppSetting.JaaSPrivateKey);
+		provider.jaasApiKey = await settings.getValueById(AppSetting.JitsiApplicationId);
 
 		return true;
 	}
 
+	async onPreSettingUpdate(context: ISettingUpdateContext, configurationModify: IConfigurationModify, read: IRead, http: IHttp): Promise<ISetting> {
+		if (context.newSetting.id === AppSetting.JitsiTokenAuditor) {
+			if (this.getProvider()?.useJaaS) {
+				context.newSetting.value = 'jitsi';
+			}
```

Comment:
What's this for?

Reply:
8x8 requires the audience to always be jitsi, figured we then make sure it is reflected in the setting always.

Reply:
Does the UI handle this properly? As in showing the new value on the setting field automatically when the settings are saved.

Reply:
probably on a rerender, not immediately. I did not check

Reply:
Tried it here and this "jitsi" value only gets applied if the user tries to set something else to the auditor setting, it doesn't get applied automatically when enabling JaaS which I assume was the intended behavior.

I believe it's best to take this out and just document this requirement.